x/subscription/keeper/cu_tracker.go

```diff
@@ -135,6 +136,12 @@ func (k Keeper) RewardAndResetCuTracker(ctx sdk.Context, cuTrackerTimerKeyBytes
 
 	totalTokenAmount := timerData.Credit.Amount
 	if totalTokenAmount.Quo(sdk.NewIntFromUint64(totalCuTracked)).GT(sdk.NewIntFromUint64(LIMIT_TOKEN_PER_CU)) {
+		remainder := sdk.NewCoin(k.stakingKeeper.BondDenom(ctx), totalTokenAmount.Quo(sdk.NewIntFromUint64(totalCuTracked)).Sub(sdk.NewIntFromUint64(LIMIT_TOKEN_PER_CU)))
```

Comment:
this calculation doesn't look right

Reply:
you're right. fixed here: https://github.com/lavanet/lava/commit/b703a4e011435a300624b7ce3250e71141488afe